src/Core/Resources/DisplayStrings.resx

```diff
@@ -0,0 +1,186 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Axe.Windows.Core.dll will need to be included for localization with these changes

Reply:
Yup, I have made note of that, thanks!